projects/plugins/jetpack/extensions/blocks/blogroll/blogroll-item/editor.scss

```diff
@@ -18,7 +17,7 @@
 	}
 
 	h3.jetpack-blogroll-item-title.rich-text {
-		color: var(--gray-gray-100, #101517);
+		color: inherit;
```

Comment:
Do we need to explicitly inherit?

Reply:
If i don't specify it, the color setting doesn't apply to the title in the editor.